packages/ses/src/compartment-shim.js

```diff
@@ -21,6 +21,7 @@ import { isValidIdentifierName } from './scope-constants.js';
 import { sharedGlobalPropertyNames } from './whitelist.js';
 import { getGlobalIntrinsics } from './intrinsics.js';
 import { tameFunctionToString } from './tame-function-tostring.js';
+import { InertCompartment, InertStaticModuleRecord } from './inert.js';
```

Comment:
I could go so far as to move all of the inert functions here, but these two were sufficient for the needs of layering.

Reply:
There are not any other iterts you could pleasantly move there. The only other `InertFoo`s are varieties of `*Function` constructors. All of these need to be made by `repairFunction`.

So this is fine as is.